src/components/molecules/ChatList/ChatList.tsx

```diff
@@ -84,20 +87,26 @@ const ChatList: React.FC<ChatListProps> = ({
   const hideDeleteModal = useCallback(() => setShowDeleteModal(false), []);
 
   const hasMessages = hasElements(messages);
+
   const messageSender = usersById?.[messageToDelete?.from ?? ""]?.partyName;
+
+  // Last (newest) message goes first
+  const sortedMessages = sortBy(messages, ["ts_utc"]).reverse();
+
   return (
     <>
       {hasMessages && (
         <div className="chat-messages-container">
           {usersById &&
-            messages.map((message, index) => (
+            sortedMessages.map((message, index) => (
               <ChatMessage
                 key={`chat-message-${index}`}
```

Comment:
Is there something more unique/good that we can use here in place of index? Maybe the `ts_utc` + who sent it or similar if messages don't have an actual UID?

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/921/commits/f69bd1a816115991a5ca4257830f370237e55697 though see also https://github.com/sparkletown/sparkle/pull/921#discussion_r537551003